c/build_all/linux/setup.sh

```diff
@@ -12,7 +12,7 @@ repo_name_from_uri()
 }
 
 scriptdir=$(cd "$(dirname "$0")" && pwd)
-deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git"
+deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake unzip openjdk-7-jre"
```

Comment:
TBH I have no idea how to build it without some of its dependencies. I found the problem that with the currently listed dependencies it doesn't build, and I fixed it so that at least it's possible to use.